wallet.py

```diff
@@ -129,7 +132,8 @@ def createFromMnemonic(words, path, password, chain):
             chain (obj): Network parameters.
 
         Returns:
-            Wallet: A wallet initialized from the seed parsed from `words`.
+            Wallet (obj): A wallet initialized from the seed parsed from
+            `words`.
```

Comment:
Either way is fine here. I generally use the `type: description` notation and only specify additional type details for lists. Your way is good too.

Reply:
I was thinking of it like `name (type): description`, same as Args:, which may be more useful in a multiple return function...